Sources/LinnaCore/Linna+Outputs.swift

```diff
@@ -0,0 +1,74 @@
+//
+//  Linna+Outputs.swift
+//  Linna
+//
+//  Created by Suita Fujino on 2018/11/12.
+//  Copyright © 2018 Suita Fujino. All rights reserved.
+//
+
+/**
+ The extension for outputs.
+ */
+extension Linna {
+    /**
+     Outputs a log message to the console.
+     
+     - Parameters:
+       - objects: Main contents for logging.
+       - filePath: The file path from which this method is called. Given by default.
+       - function: The function name from which this method is called. Given by default.
+       - line: The number of line from which this method is called. Given by default.
+     */
+    public static func cout(_ objects: Any..., filePath: String = #file, functionName: String = #function, lineNumber: Int = #line) {
+        output(objects, with: consoleStream, filePath: filePath, functionName: functionName, lineNumber: lineNumber)
+    }
+    
+    /**
+     Outputs a log message to the file.
+     
+     - Parameters:
+     - objects: Main contents for logging.
+     - filePath: The file path from which this method is called. Given by default.
+     - function: The function name from which this method is called. Given by default.
+     - line: The number of line from which this method is called. Given by default.
+     */
+    public static func fout(_ objects: Any..., filePath: String = #file, functionName: String = #function, lineNumber: Int = #line) {
+        #if DEBUG
+        if let localFileStream = localFileStream {
+            output(objects, with: localFileStream, filePath: filePath, functionName: functionName, lineNumber: lineNumber)
+        } else {
+            iPrint(message: "File stream is not ready. Log messages for the file stream will be discarded.")
+        }
+        #endif
+    }
+    
+    /**
+     Outputs a log message with the explicit stream.
+     
+     - Parameters:
+     - objects: Main contents for logging.
+     - stream: The output stream confirms to `LinnaStream`.
+     - filePath: The file path from which this method is called. Given by default.
+     - function: The function name from which this method is called. Given by default.
+     - line: The number of line from which this method is called. Given by default.
+     */
+    public static func output(_ objects: Any..., with stream: LinnaStream, filePath: String = #file, functionName: String = #function, lineNumber: Int = #line) {
+        // TODO: Make settable
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Scior/Linna/blob/cbea4ce0c9f9983bbb8744e6511f07789db98d4f/Sources/LinnaCore/Linna+Outputs.swift#L56"></span>TODOs should be resolved (Make settable).</td>
    </tr>
  </tbody>
</table>